exercises/practice/robot-name/src/test/java/RobotTest.java

```diff
@@ -34,6 +37,17 @@ public void resetName() {
         assertThat(name).isNotEqualTo(name2);
         assertIsValidName(name2);
     }
+    
+    @Ignore("Remove to run test")
+    @Test
+    public void robotNamesAreUnique() {
+        Set<String> robotNames = new HashSet<>();
+        Integer sampleSize = 5000;
```

Comment:
Super small nitpicky thing: use int rather than Integer here. Why? Because Integer makes it seem like you care about using the boxed value (eg. you might need null or something), but you do not in this case.

```suggestion
        int sampleSize = 5000;
```